index.html

```diff
@@ -17,14 +17,14 @@
 			<img class="world" src="images/world.png">
 		</div>
 	</header>
-	<div class="intro">
+	<div id="intro-container" class="intro">
```

Comment:
Added for consistency, and because we probably want to hide/show the whole intro, and not just the button.